src/test/java/com/williamfiset/algorithms/sorting/RadixSortTest.java

```diff
@@ -0,0 +1,27 @@
+package com.williamfiset.algorithms.sorting;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.Test;
+
+public class RadixSortTest {
+
+  @Test
+  public void testGetMax() {
+    int[] array = {3, 90, 112, 56};
+    int[] array2 = {5, 9, 1, 42, 117};
+    int max = RadixSort.getMax(array);
+    int max2 = RadixSort.getMax(array2);
+    assertEquals(112, max);
+    assertEquals(117, max2);
+  }
+
```

Comment:
Can you add more test coverage? Can you also add some randomized tests? E.g:

https://github.com/williamfiset/Algorithms/blob/master/src/main/java/com/williamfiset/algorithms/sorting/SelectionSort.java#L45